.scripts/git-branch.ps1

```diff
@@ -0,0 +1,74 @@
+[CmdletBinding()]
+param
+(
+    [Parameter(Mandatory = $true)]
+    [string]$Name,
+    
+    [Parameter(Mandatory = $true)]
+    [string]$Email
+)
+
+function TagExists($TagName) {
+    
+    git tag | ForEach-Object {
+
+        if ($_ -eq $TagName) {
+            return $true
+        }
+    }
+
+    return $false
+}
+
+$date = Get-Date -UFormat "%Y-%m-%d"
+$branch = "release_$date"
+
+$sources = $env:BUILD_SOURCESDIRECTORY
+
+Write-Output "Set current directory to '$sources'"
+Set-Location $sources
+
+git config user.email $Email
+git config user.name $Name
+
+$message = "Bump package versions for release $date"
+
+Write-Output "Create new branch '$($branch)'"
+
+git checkout -b $branch
+
+$lastCommitId = git rev-list --tags --max-count=1
+$lastTag = git describe $lastCommitId  # By default git describe only shows annotated tags.
+
+if ($LASTEXITCODE -ne 0) {
+
+    git checkout master
+    git branch -d
+
+    throw "Failed to find last tagged commit ($LASTEXITCODE)"
+}
+
+Write-Output "Print commits from $lastTag..$branch"
+git log $lastTag..$branch --name-status
+
+git add .
+git commit -m  $message
```

Comment:
i'm not seeing any error checking in here. maybe let's not block to check this in but i'm worried about error piling on if one of the first commands fail and then we check in everything.